_config.yml

```diff
@@ -55,5 +55,7 @@ footer_links:
   - url: https://www.facebook.com/TechWorkersCoalition
     text: Facebook
     newTab: true
+sass:
```

Comment:
Thank you for taking care of this! One last thing is to remove these 3 lines, because they are front matters, a thing only Jekyll uses https://github.com/techworkersco/twc-site/blob/7465dda8a24a3fc75b84c98906f954865a78412c/_includes/main.scss#L1-L3

Reply:
I've finished the requested changes
Thank you so much for your valuable comments